uniffi_macros/src/enum_.rs

```diff
@@ -89,14 +89,14 @@ pub(crate) fn enum_ffi_converter_impl(
 
     quote! {
         #[automatically_derived]
-        impl ::uniffi::RustBufferFfiConverter for #ident {
-            type RustType = Self;
+        unsafe impl ::uniffi::FfiConverter<crate::UniFFITag> for #ident {
```

Comment:
The derive always has its code expanded in the same scope as the type defonition, so it can and should be a blanket impl I think:


```suggestion
        unsafe impl<T> ::uniffi::FfiConverter<T> for #ident {
```